### ISSUE TITLE
ZAP window takes focus when a tab list item is selected or the Request/Response forms are refreshed

### ISSUE THREAD

**Issue by fallenby** [zaproxy/zaproxy#1844]
I'm using multiple monitors; one with ZAP on it and the other with my browser. When I have an item in the History/other list selected, ZAP steals focus from the web browser and forces my cursor onto the other monitor.

This also happens when a breakpoint is hit despite having the "bring to front" option disabled.

As it stands, ZAP is very annoying to use with multiple monitors like this, as the History is often changed many, many times, and prevents me from using anything other than ZAP while something is happening.

This may also happen in other instances though I haven't tested everything.

**Comment by psiinon** [comment]
Yeah, we should have an option to disable that (whatevers causing it)

**Comment by fallenby** [comment]
I can confirm that this happens on stock Ubuntu with Unity.
